apps/ledger-live-mobile/src/newArch/features/MemoTag/hooks/useMemoTagInput.ts

```diff
@@ -17,22 +17,23 @@ export const useMemoTagInput = (
       (perFamily[family as keyof typeof perFamily] as FC<MemoTagInputProps>)) ||
     null;
 
+  const [isProcessing, setIsProcessing] = useState(false);
   const [isEmpty, setIsEmpty] = useState(true);
   const [error, setError] = useState<Error | undefined>();
-  const debouncedUpdateTransaction = useMemo(
-    () => debounce(updateTransaction, DEBOUNCE_DELAY),
-    [updateTransaction],
-  );
-  const handleChange = useCallback<MemoTagInputProps["onChange"]>(
-    ({ patch, value, error }) => {
+  const handleChange = useMemo<MemoTagInputProps["onChange"]>(() => {
```

Comment:
why `useMemo` instead of `useCallback` ?

Reply:
It's just to keep `debouncedUpdateTransaction` in the scope of the function. IMO it is simpler to have the whole thing have only `updateTransaction` as it's dependency rather than having `debouncedUpdateTransaction` as a dependency of `handleChange` which itself depends on `updateTransaction`.

Reply:
ok i understand your point.